features/guidance/low-priority.feature

```diff
@@ -0,0 +1,98 @@
+@routing  @guidance
+Feature: Exceptions for routing onto low-priority roads
+
+    Background:
+        Given the profile "car"
+        Given a grid size of 10 meters
+
+    Scenario: Straight onto low-priority: same name
+        Given the node map
+            """
+                c
+
+            a   b d
+
+                e
+            """
+
+        And the ways
+            | nodes  | highway     | name    |
+            | abd    | residential | road    |
+            | eb     | service     | service |
+            | bc     | service     | service |
+
+       When I route I should get
+            | waypoints | route            | turns         |
+            | c,e       | service,service  | depart,arrive |
+            | e,c       | service,service  | depart,arrive |
+
+    Scenario: Straight onto low-priority: onto and from unnamed
+        Given the node map
+            """
+                c
+
+            a   b d
+
+                e
+            """
+
+        And the ways
+            | nodes  | highway     | name    |
+            | abd    | residential | road    |
+            | eb     | service     | service |
+            | bc     | service     |         |
+
+       When I route I should get
+            | waypoints | route             | turns                       |
+            | e,c       | service,          | depart,arrive               |
+            | c,e       | ,service,service  | depart,turn straight,arrive |
```

Comment:
just for clarification: this is due to the name not being the same anymore? I'd probably still see the exact same shape here as obvious, but having this turn in here or not is much more a personal preference. So feel free to keep it this way.